src/app/loadout-builder/filter/FilterBuilds.tsx

```diff
@@ -77,7 +77,7 @@ export default function FilterBuilds({
           </label>
           <RangeSelector
             min={750}
-            max={parseInt(selectedStore.stats.maxGearPower!.value.toString(), 10)}
+            max={parseInt(selectedStore.stats.maxGearPower?.value.toString() ?? '750', 10)}
```

Comment:
https://sentry.io/share/issue/639a0aa388a94fb2861bad11be1b7216/

Reply:
This is super weird, how can a store not have maxGearPower?